Raspi/Test/mock_wireless_module.py

```diff
@@ -44,6 +44,8 @@
                    ("longitude", 25),
                    ("altitude", 50),
                    ("course", 0))
+s_power = MockSensor(200, percent_range=0.25)
```

Comment:
nit: Increase the percent range to like 80%? Would be good to be able to change how these random numbers are generated one day.